cas/cas.go

```diff
@@ -59,9 +61,10 @@ var diskvStores = [...]string{
 
 // Store encapsulates a content-addressable-storage for storing ACIs on disk.
 type Store struct {
-	base   string
-	stores []*diskv.Diskv
-	db     *DB
+	base      string
+	stores    []*diskv.Diskv
+	db        *DB
+	treestore *TreeStore
```

Comment:
For overlayfs I need either the treestore exported or Check() (to make sure the tree is in a good state before mounting/preparing the image) and GetRootFS() (to get the lower overlayfs dir) functions exposed through Store.